docs/guides/rookieguide.md

```diff
@@ -2,6 +2,105 @@
 
 ## Wstęp
 
-Poradnik został opracowany z myślą o graczach dopiero rozpoczynających członkostwo w naszej grupie - jego zamiarem jest poprowadzenie was przez rękę przez wszystkie meandry konfigurowania Army i radzenia sobie z wypracowanymi wewnątrz grupy systemami, do których należą między innymi zasady zapisywania się na misję. Poszczególne kroki są dokładnie opisane, te bardziej wymagające popieramy zrzutami ekranów, znajdziecie w nim również podstawowe FAQ dotyczące zagadnień ogólnych. W razie jakichkolwiek problemów ze zrozumieniem treści poradnika kontaktujcie się ze swoimi opiekunami Rekruterami, lub uderzajcie do jego twórcy na Discordzie - @Nakimeteles.
+Poradnik został opracowany z myślą o graczach dopiero rozpoczynających członkostwo w naszej grupie - jego zamiarem jest poprowadzenie ciebie za rękę przez wszystkie meandry konfigurowania Army i radzenia sobie z wypracowanymi wewnątrz grupy systemami, do których należą między innymi zasady zapisywania się na misję. Poszczególne kroki są dokładnie opisane, te bardziej wymagające popieramy zrzutami ekranów, znajdziecie w nim również podstawowe FAQ dotyczące zagadnień ogólnych. W razie jakichkolwiek problemów ze zrozumieniem treści poradnika skontaktuj się ze swoimi opiekunami Rekruterami.
 
-[Przejdź do poradnika dla rekrutów](https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit?usp=sharing)
+## *Discord Arma Forces*
+
+### 1. Wejdź na Discord Arma Forces.
```

Comment:
Consider adding `markdownlint` and `Markdown All in One` extensions to Visual Studio Code, it'll highlight lines which aren't fully compliant with how Markdown docs are created. I see there are 5 rules violated in this file:
- MD026/no-trailing-punctuation: Trailing punctuation in heading [Punctuation: '.']
  - Multiple headers with this issue (with one fix suggestion here)
- MD009/no-trailing-spaces: Trailing spaces [Expected: 0 or 2; Actual: 1]
- MD032/blanks-around-lists: Lists should be surrounded by blank lines
  - In `Importowanie modlisty` there is no blank line before numbered list.
- MD026/no-trailing-punctuation: Trailing punctuation in heading [Punctuation: '!']
  - One of questions has `!` at the end.
- MD036/no-emphasis-as-heading/no-emphasis-as-header: Emphasis used instead of a heading
  - Last line of the file, whole line has emphasis and it thinks it's a header
- MD034/no-bare-urls: Bare URL used
  - This is for the last link in this file, it's just plaintext instead of `[<description>](<link)`
```suggestion
### 1. Wejdź na Discord Arma Forces
```